tools/hist/fiologparser_hist.py

```diff
@@ -473,7 +473,7 @@ def main(ctx):
         try:
             from configparser import SafeConfigParser, NoOptionError
         except ImportError:
-            from ConfigParser import SafeConfigParser, NoOptionError
+            from configparser import SafeConfigParser, NoOptionError
```

Comment:
If we drop py2 we can assume we can do this and drop the try...catch.

Reply:
Not sure whether we can drop Python 2 support ...

Reply:
If we are keeping python2 I'd argue the above is wrong as the module is called `ConfigParser` (note the case) in python2. However I still think we should convert all scripts over to python3 and call it a day...

Those on (older) CentOS can grab python3 via Software Collections (https://developers.redhat.com/blog/2018/08/13/install-python3-rhel/ ) and since fio already has some scripts that are python 3 only this seems reasonable. At any rate, base fio itself doesn't use python so if that's all people were after then there's no change from their perspective...